server.js

```diff
@@ -8,18 +9,21 @@ const { getTodaysChores } = require("./utilities/getChores.js");
 const { sendNoChores, postToSlack } = require("./utilities/notify.js");
 const { markChoreDone } = require("./utilities/markChoreDone.js");
 
-//package configuration
-
-//local configuration
-
-//globals
-const CRON_SCHEDULE = process.env.CRON_SCHEDULE;
```

Comment:
Why did you remove this here rather than adding further globals? In other files, we make globals from `process.env` variables; here, we're not doing it. I prefer seeing all the globals in one place at the top of the file.

Reply:
If you use environment variables directly instead of caching them in variables, it allows for the variables to be changed while the code is still running.

Reply:
Thank you for mentioning this; I had not thought about it. You make a good point, except that that doing this would most often require the container to be `--force-recreate`d as the envs are stored in `docker-compose.yml`.

Reply:
Fair point; however, I believe it is not necessary to store an environment variable in memory as a constant if it does not need a default value specified in the code.

Reply:
I do realize doing it the way I mentioned is the storage of an extra var in memory.

Reply:
^ sounds good. I'll approve.